src/sentry/static/sentry/app/components/idBadge/baseBadge.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Avatar from 'app/components/avatar';
+import space from 'app/styles/space';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import {Team, Member, User, Organization, AvatarProject} from 'app/types';
+
+type Props = {
+  displayName: React.ReactNode;
```

Comment:
Nit: This should be moved between `project` and `description`.

Reply:
@leedongwei please why do you believe that this should be moved? I usually follow the convention, adding the optional props in the last positions of the object...